compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/ComposeScene.skiko.kt

```diff
@@ -583,13 +595,22 @@ class ComposeScene internal constructor(
     }
 
     private fun processPress(event: PointerInputEvent) {
-        val owner = hoveredOwner(event)
-        if (focusedOwner.isAbove(owner)) {
-            focusedOwner?.onDismissRequest?.invoke()
-        } else {
-            owner?.processPointerInput(event)
-            pressOwner = owner
+        val owner = reversedOwners {
```

Comment:
Why not `forEachOwnerReversed`? I don't mean the name, but that the action can be passed the owners, not the list.

Reply:
Because we need "break" logic here